packages/file-search/src/node/file-search-service-impl.ts

```diff
@@ -33,6 +33,10 @@ export class FileSearchServiceImpl implements FileSearchService {
         @inject(RawProcessFactory) protected readonly rawProcessFactory: RawProcessFactory) { }
 
     async find(searchPattern: string, options: FileSearchService.Options, clientToken?: CancellationToken): Promise<string[]> {
+        if (searchPattern === "") {
```

Comment:
@RDIL please use singlequotes for strings (reason for the build failure).

https://github.com/eclipse-theia/theia/blob/86704a9ca6106ca5c9c34f7b42c6066b82cb1d82/configs/errors.eslintrc.json#L7-L13

Reply:
Ah got it

Reply:
Fixed.